signal/cosmos.go

```diff
@@ -52,6 +54,9 @@ func (c *Cosmos) getHeaders() map[string]string {
 }
 
 func (c *Cosmos) getEndpoints() []string {
+	if len(c.Endpoints) != 1 {
+		log.Errorf("Cosmos needs 1 endpoint, got %d", len(c.Endpoints))
```

Comment:
We can't fail out here since we want to do a best effort if other reporters are present. Best case is we alert that the reporter didn't have any endpoints, which would point to a misconfigured config file.